runelite-client/src/main/java/net/runelite/client/plugins/bank/BankItemsOverlay.java

```diff
@@ -0,0 +1,166 @@
+import net.runelite.api.Item;
+import net.runelite.client.game.ItemManager;
+import net.runelite.client.ui.overlay.Overlay;
+import net.runelite.client.ui.overlay.OverlayPosition;
+import net.runelite.client.ui.overlay.components.ComponentOrientation;
+import net.runelite.client.ui.overlay.components.ImageComponent;
+import net.runelite.client.ui.overlay.components.PanelComponent;
+import net.runelite.client.ui.overlay.components.TitleComponent;
+import net.runelite.client.util.StackFormatter;
+
+import javax.inject.Inject;
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.Graphics2D;
+import java.awt.Point;
+import java.awt.image.BufferedImage;
+import java.util.Arrays;
+import java.util.Comparator;
+
+@Slf4j
+class BankItemsOverlay extends Overlay
+{
+	private static final int ITEM_SIZE = 10;
+	private static final int PLACEHOLDER_WIDTH = 36;
+	private static final int PLACEHOLDER_HEIGHT = 32;
+	private static final int WRAPPING = 2;
+	private static final Point GAP = new Point(10, 6);
+	private static final ImageComponent PLACEHOLDER_IMAGE = new ImageComponent(new BufferedImage(PLACEHOLDER_WIDTH, PLACEHOLDER_HEIGHT, BufferedImage.TYPE_4BYTE_ABGR));
+
+	private final PanelComponent panelComponent = new PanelComponent();
+
+	private final ItemManager itemManager;
+	private final BankConfig config;
+	private final BankCalculation bankCalculation;
+
+
+	private ItemPrice[] expensiveItems;
+
+	@Inject
+	private BankItemsOverlay(ItemManager itemManager, BankConfig config, BankCalculation bankCalculation)
+	{
+		setPosition(OverlayPosition.BOTTOM_RIGHT);
+		panelComponent.setWrapping(WRAPPING);
+		panelComponent.setGap(GAP);
+		panelComponent.setOrientation(ComponentOrientation.HORIZONTAL);
+		panelComponent.setPreferredSize(new Dimension(PLACEHOLDER_WIDTH + GAP.x, PLACEHOLDER_HEIGHT + GAP.y));
+
+		this.itemManager = itemManager;
+		this.config = config;
+		this.bankCalculation = bankCalculation;
+	}
+
+	@Override
+	public Dimension render(Graphics2D graphics)
+	{
+		if (!config.showMostExpensiveItemsOverlay())
+		{
+			return null;
+		}
+
+		final Item[] bankItems = bankCalculation.getBankItems();
+
+		if (bankItems == null || bankItems.length == 0)
+		{
+			return null;
+		}
+
+		panelComponent.getChildren().clear();
+
+		if (expensiveItems == null || bankCalculation.isBankDifferent(bankItems)) 
+		{
+			expensiveItems = getExpensiveItems(bankItems);
+			Arrays.sort(expensiveItems, Comparator.comparing(ItemPrice::getPrice).reversed());
+		}
+
+
+		for (int i = 0; i < ITEM_SIZE; i++)
+		{
+			if (i < expensiveItems.length)
+			{
+				final ItemPrice itemPrice = expensiveItems[i];
+				if (itemPrice.getItem().getQuantity() > 0)
+				{
+					final BufferedImage image = getImage(itemPrice.getItem());
+
+					if (image != null)
+					{
+						ImageComponent imageComponent = new ImageComponent(image);
+						panelComponent.getChildren().add(imageComponent);
+
+						TitleComponent titleComponent = TitleComponent.builder()
```

Comment:
Why is this a title component

Reply:
Why not?

As I mentioned above, UI isn't a strong-suit of mine. In fact, I have zero experience with Java AWT other than this change. With that said, I think it would be more helpful to explain why you think a `TitleComponent` isn't the correct solution here and which alternative you prefer. That would really help me out, along with any other readers with similarly little experience with the code base.